scripts/postversion.sh

```diff
@@ -4,7 +4,9 @@ set -eu
 
 cd "$(dirname "$0")"
 
-git push https://github.com/hypothesis/client.git master:master --follow-tags
+# nb. The remote refname is fully qualified because this script is run in a CI
+# environment where not all heads may have been fetched.
+git push https://github.com/hypothesis/client.git HEAD:refs/heads/$BRANCH_NAME --follow-tags
```

Comment:
`BRANCH_NAME` comes from the Docker environment where the build is run. See the `Jenkinsfile`.